cedar-discord/src/discord.cedarschema

```diff
@@ -0,0 +1,19 @@
+// discord permissions, (e.g. SendMessage)
+entity Permission;
```

Comment:
It seems `Permission`s overlap `Action`s. I think we don't need this entity type.